pages/styles/index.css

```diff
@@ -0,0 +1,82 @@
+* {
+    font-family: sans-serif;
+}
+
+.links {
+    position: relative;
+    left: 430px; 
+    width: 70%;
+    height: 35vh;
+    background: white;
+    font-size: 25px;
+} 
+
+nav {
+    display: flex;
+    width: 100%;
+    padding: 30px 0;
+    align-items: center;
+    justify-content: space-between;
+} 
+
+header a {
+    margin-left: 640px;
+}
+
+ul li {
+    display: inline-block;
+    list-style: none;
+}
+
+ul, li, a {
+    text-decoration: none;
+    color: #c6fbf9;
+    font-weight: bold;
+}  
```

Comment:
While having separate styles for each page is fine, it does lead to some cases where rules are repeated between pages. For example, this `ul, li, a` rule is repeated in portfolio.css. (I'm sure there's other cases but I didn't track them all down.)

You _can_ have multiple stylesheets loaded on a page, so you could move the common rules to perhaps a common.css file, and include both files. In fact, if common.css was included first, you can then slightly override the rules with your page-specific file. This is part of the whole "cascading" functionality of CSS!